Update Range.cloneContents()

myakura opened the pull request:
* adds xref to the `cloneNode()` method
* clarify "using DOM Events" by xrefing to the `addEventListener()`
* wrap "id" using `<code></code>` in "HTML id attributes"
* remove`em` in the syntax section

> Issue number to be fixed (ie 'Fixes #123', if there is an associated issue)

none.


> What was wrong/why is this fix needed? (quick summary only)

for xref updates, I felt it is helpful to have those links.


> Anything else that could help us review it

myakura committed:
update Range.cloneContents()

* adds xref to the `cloneNode()` method
* clarify "using DOM Events" by xrefing to the `addEventListener()`
* wrap "id" using `<code></code>` in "HTML id attributes"

myakura committed:
update Range.cloneContents()

replace em in the syntax section with var

SVarvello commented:
hacker on myvphone

On Sat, Jul 31, 2021, 5:49 PM github-actions[bot] ***@***.***>
wrote:

> Preview URLs
>
>    -
>    https://pr7477.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/Range/cloneContents
>
> Flaws
>
> *None!* 🎉
> External URLs
>
> URL: /en-US/docs/Web/API/Range/cloneContents
> <https://pr7477.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/Range/cloneContents>
> Title: Range.cloneContents()
> on GitHub
> <https://github.com/mdn/content/blob/7477/merge/files/en-us/web/api/range/clonecontents/index.html>
>
> No *new* external URLs
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mdn/content/pull/7477#issuecomment-890424726>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUVOB4MH5WITD5U3ZQKALITT2SK2LANCNFSM5BKRKC5Q>
> .
>

sideshowbarker commented on files/en-us/web/api/range/clonecontents/index.html:
```diff
@@ -14,16 +14,17 @@
 <h2 id="Syntax">Syntax</h2>
 
-<pre class="brush: js"><em>documentFragment </em>= <em>range</em>.cloneContents();
+<pre class="brush: js"><var>documentFragment</var> = <var>range</var>.cloneContents();
```
```suggestion
<pre class="brush: js">documentFragment = range.cloneContents();
```
Any markup inside `pre` elements won’t survive HTML➡Markdown conversion, so we might as well go ahead and just remove it.

myakura replied:
thanks!

myakura committed:
update Range.cloneContents()

remove var tags in syntax code as it won't be kept once converted to markdown.
https://github.com/mdn/content/pull/7477#discussion_r680442434
